### ISSUE TITLE
No models cast shadows

### ISSUE THREAD

**Issue by MagicalPaladin** [armory3d/armory#111]
As the title says...
![armorynoshadows](https://cloud.githubusercontent.com/assets/15276930/21783113/4f6e421c-d6b6-11e6-89a4-139b878119d9.png)

**Comment by luboslenco** [comment]
Do you still have .blend file for this somewhere?

**Comment by MagicalPaladin** [comment]
Nope, sorry, will try with new build to see if it's doing the same issue

**Comment by ja-moli** [comment]
Yesterday I worked several times with Armory and I discovered that happened what you're showing in the image. No shadows are casted, and also the lamps doesn't emit light with force.
It simply seemed that the lighting engine was not working at all. After several closes and opening of the entire program and several renderings (needed for the tests I was making) the lihgts began to work properly. I think its related with graphic card (or drivers). Like the card not knowing that the lights must be rendered.
